src/modules/ud/components/content-type-selector/content.type.selector.js

```diff
@@ -0,0 +1,74 @@
+import React, { useState, useContext } from 'react';
+
+import { createCssClassNames } from '../../../common/helpers/css.class.names';
+import { SelectedContentTypesContext } from '../search/search';
+
+const ContentTypeSelector = () => {
+    const { contentTypes: contentTypesMap } = window.eZ.adminUiConfig;
+    const [selectedContentTypes, dispatchSelectedContentTypesAction] = useContext(SelectedContentTypesContext);
+    const [collapsedGroups, setCollapsedGroups] = useState(() => {
+        return Object.keys(contentTypesMap).reduce((collapsedGroups, contentTypeGroup, index) => {
+            return { ...collapsedGroups, [contentTypeGroup]: !!index };
+        }, {});
+    });
+    const toggleCollapsed = ({ nativeEvent }) => {
+        const contentTypeGroup = nativeEvent.target.dataset.id;
+
+        setCollapsedGroups((prevState) => {
+            return { ...prevState, [contentTypeGroup]: !prevState[contentTypeGroup] };
+        });
+    };
+    const handleContentType = ({ nativeEvent }) => {
```

Comment:
This name does not tell what this function is handling. I would change it to something like: `handleContentTypeSelect` etc.